### PR TITLE
Answer some DevOps questions

### PR DESCRIPTION
**Pull request by cloudgrimm**
Also did a minor grammar change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,10 +133,14 @@ Each piece of code (change/patch) is verified, to make the change is safe to mer
 
 <details>
 <summary>What is Continuous Deployment?</summary><br><b>
+
+A development strategy used by developers to release software automatically into production where any code commit must pass through an automated testing phase. Only when this is successful is the release considered production worthy. This eliminates any human interaction and should be implemented only after prodcution-ready pipelines have been set with real-time monitoring and reporting of deployed assets. If any issues are detected in production it should be easy to rollback to previous working state.
```
Comment (bregman-arie) [inline README.md:137]: Did you wrote it by yourself or copied it from somewhere? If copied, please mention in the answer where the answer is taken from (with a link). Thanks.

**Review by bregman-arie — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (CloudGrimm) [reply README.md]: Not answered entirely by myself but will insert link with the long explanation.

File: README.md [hunk 2]
```diff
@@ -133,10 +133,18 @@ Each piece of code (change/patch) is verified, to make the change is safe to mer
 
 <details>
 <summary>What is Continuous Deployment?</summary><br><b>
+
+A development strategy used by developers to release software automatically into production where any code commit must pass through an automated testing phase. Only when this is successful is the release considered production worthy. This eliminates any human interaction and should be implemented only after production-ready pipelines have been set with real-time monitoring and reporting of deployed assets. If any issues are detected in production it should be easy to rollback to previous working state. 
+
+For more info please read [here](https://searchitoperations.techtarget.com/definition/continuous-deployment#:~:text=Continuous%20deployment%20is%20a%20strategy,visible%20to%20the%20software's%20users.)
```
Comment (bregman-arie) [inline README.md:139]: hmm maybe a site that doesn't require you to sign in?

**Review by bregman-arie — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (CloudGrimm) [reply README.md]: chnaged

**Pull request merged**